src/libAtomVM/bif.h

```diff
@@ -39,11 +39,20 @@ extern "C" {
 
 #define MAX_BIF_NAME_LEN 260
 
+struct SubBinarySlice
```

Comment:
We are using this struct only as an output of `get_sub_binary_slice`, so we might link better the 2 names when we rename the function. It might be called `SlicedBinary` or `SanitizedSlice` or whatever we'll use as new name.
Also let's not forget to do the typedef trick, so we don't have to use all around **`struct`**`SubBinarySlice foo_bar;`